core/src/main/java/org/pentaho/di/connections/vfs/VFSConnectionProvider.java

```diff
@@ -57,4 +57,29 @@ default FileObject getDirectFile( T connectionDetails, String path ) throws Kett
     return KettleVFS.getFileObject( pvfsUrl, new Variables(), getOpts( connectionDetails ) );
   }
 
+  /**
+   * Test the VFS connection with provided connection details.
```

Comment:
Doclet is not documenting the parameters...

See related comment: https://github.com/pentaho/pentaho-kettle/pull/9305#discussion_r1595496535.

See other ConnectionManager related comment TBD.

Should mention that the default implementation exists just calls the other version of the method.

Reply:
changes addressed.